test/functional/eval/timer_spec.lua

```diff
@@ -19,72 +19,85 @@ describe('timers', function()
   end)
 
   it('works one-shot', function()
-    command("call timer_start(50, 'MyHandler')")
-    eq(0,eval("g:val"))
-    run(nil, nil, nil, load_adjust(200))
+    eq(0, eval("[timer_start(10, 'MyHandler'), g:val][1]"))
+    run(nil, nil, nil, load_adjust(100))
     eq(1,eval("g:val"))
   end)
 
   it('works one-shot when repeat=0', function()
-    command("call timer_start(50, 'MyHandler', {'repeat': 0})")
-    eq(0,eval("g:val"))
-    run(nil, nil, nil, load_adjust(200))
-    eq(1,eval("g:val"))
+    eq(0, eval("[timer_start(10, 'MyHandler', {'repeat': 0}), g:val][1]"))
+    run(nil, nil, nil, load_adjust(100))
+    eq(1, eval("g:val"))
   end)
 
-
   it('works with repeat two', function()
-    command("call timer_start(50, 'MyHandler', {'repeat': 2})")
-    eq(0,eval("g:val"))
```

Comment:
the reason for assertions like this is to make sure the editor doesn't accidentally have some variable defined that makes the test pass. (could be from a previous non-cleared session for example, or some new plugin, etc.)

Reply:
It is included still, but in a single call: `eq(0, eval("[timer_start(10, 'MyHandler', {'repeat': 2}), g:val][1]"))` - to avoid flakiness due to this.  I.e. it got increased already before the check then (seen initially with timeouts of 50ms IIRC)